web/content/release-3.6.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 3.6
+---
+:FOREMAN_VER: 3.6
+:KATELLO_VER: 4.8
+
+== Foreman {FOREMAN_VER} and Katello {KATELLO_VER}
+
+Use the top menu bar for navigation for all guides.
+
+=== Quickstart
+
+* link:/{FOREMAN_VER}/Quickstart/index-katello.html[Katello on RHEL/CentOS Quickstart Guide]
+
+=== Installation
+
+* link:/{FOREMAN_VER}/Installing_Server/index-katello.html[Installing Katello on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Proxy/index-katello.html[Installing Smart Proxy with content on RHEL/CentOS]
+
+=== Upgrading
+
+* link:/{FOREMAN_VER}/Release_Notes/index-katello.html[Release notes for Katello on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Upgrading_and_Updating/index-katello.html[Upgrading and Updating Katello on RHEL/CentOS]
```

Comment:
On a second thought, this file probable should have `.adoc` suffix.

Reply:
Good eye! Thanks!